ci_framework/roles/edpm_build_images/tasks/cleanup.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I see you have removed this task from main.yaml, we can get rid of this file if we don't want to do any cleanup afterwards.